code/app/app.py

```diff
@@ -178,9 +178,9 @@ def conversation_with_data(request):
         return Response(json.dumps(r).replace("\n", "\\n"), status=status_code)
     else:
         if request.method == "POST":
-            return Response(stream_with_data(body, headers, endpoint))
+            return Response(stream_with_data(body, headers, endpoint), mimetype="application/json-lines")
         else:
-            return Response(None)
+            return Response(None, mimetype="application/json-lines")
```

Comment:
This is for a different PR, but shouldn't this be an HTTP 405 (Method Not Allowed)? Not sure how your UI renders it.

Reply:
Yes I'm not quite sure what the original intention was here, or why the endpoint supposedly accepts GET requests, but I thought it would be best to leave it as-is and just change the mimetypes

Reply:
I've raised it in a separate issue

https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/issues/197